USBPcapDriver/USBPcapTables.c

```diff
@@ -54,8 +54,11 @@ VOID USBPcapAddEndpointInfo(IN PRTL_GENERIC_TABLE table,
                                  &new);
 
     if (new == FALSE)
-    {
-        DkDbgStr("Element already exists in table");
+    {        
+		DkDbgStr("Element already exists in table. Lets replace it!");
+		USBPcapRemoveEndpointInfo(table, pipeInfo->PipeHandle);
```

Comment:
RtlInsertElementGenericTable function documentation remarks mention:
> If a matching element already exists in the generic table, RtlInsertElementGenericTable returns a pointer to the existing element's data and sets NewElement to FALSE.

We could save the returned pointer, and if the new is FALSE, update the existing structure information.

Besides this the source code uses 4 spaces for indentation (no tabs).